### ISSUE TITLE
Autostart with Wayland

### ISSUE THREAD

**Issue by yulipumal** [grafana/grafana-kiosk#135]
The latest grafana-kiosk version (1.0.7) runs correctly from cli (wayland and Pi 5):
`/usr/bin/grafana-kiosk -c /home/pi/infoscreens/config.yml`

But when I tried to run it on wayland autostart, it fails:
~/config/wayfire.ini
```
...
[autostart]
grafana_kiosk = /home/pi/infoscreens/startKiosk.sh
```

**Comment by briangann** [comment]
the wayland support was completed, will add the configuration file based option that will let it work.

**Comment by hashworks** [comment]
Any progress on this? I would love to use it with the cage wayland compositor.

**Comment by yulipumal** [comment]
Still not working in 1.0.7

**Comment by briangann** [comment]
yes, though v1.1.0 is going to change a lot of configuration options, hopefully migrating won't be too rough.
